assets/src/edit-story/app/media/media3p/useFetchMediaEffect.js

```diff
@@ -42,9 +43,10 @@ export default function useFetchMediaEffect({
       try {
         const { media, nextPageToken } = await listMedia({
           provider,
+          searchTerm,
           pageToken,
         });
-        fetchMediaSuccess({ provider, media, nextPageToken });
+        fetchMediaSuccess({ provider, media, pageToken, nextPageToken });
```

Comment:
pageToken doesn't need to be passed to fetchMediaSuccess, only nextPageToken. The 'setNextPage' action is responsible for assigning pageToken = nextPageToken.

Reply:
Needs to be passed so we decide if we have to append or replace, right?

Reply:
That makes sense.